firmware/library/newlib/newlib.cpp

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
This whole commit should be dropped. Will cause issues if a stdlibc call calls these functions.

Reply:
done